.rubocop.yml

```diff
@@ -74,10 +75,10 @@ Style/IndentationWidth:
 Style/IndentHash:
   Severity: fatal
   Enabled: true
+  EnforcedStyle: consistent
 
 Style/InlineComment:
-  Severity: fatal
-  Enabled: true
+  Enabled: false
```

Comment:
Again, this was a code design decision.

The rationale was that code comments are a code smell. 
Refactor your code to make code comments irrelevant.

Although I appreciate that's not _always_ possible. We still decided to disable comments because it meant it forced people to really think about the code they were writing and justifying the use of a comment